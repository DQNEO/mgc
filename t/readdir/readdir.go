package main

import (
	"fmt"
	"os"
)

func f1() {
	dir := "t/data"
	f, err := os.Open(dir)
	if err != nil {
		panic(err)
	}
	names, err := f.Readdirnames(-1)
	if err != nil {
		panic(err)
	}

	fmt.Printf("%d\n", len(names))
}

func main() {
	f1()
}
