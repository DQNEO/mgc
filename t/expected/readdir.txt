hello.go
sample.txt
string.txt
